.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+exclude: ^(.*\.proto)
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+        exclude: ^(examples|django_socio_grpc/tests/fakeapp/grpc)/
```

Comment:
why excluding django_socio_grpc ?

Reply:
It's not django_socio_grpc it's django_socio_grpc/tests/fakeapp/grpc

Same as in .flake8